python/ray/actor.py

```diff
@@ -583,45 +529,25 @@ def method(self):
 
         actor_cls = self
 
-        new_runtime_env = parse_runtime_env(runtime_env)
-
-        cls_options = dict(
-            num_cpus=num_cpus,
-            num_gpus=num_gpus,
-            memory=memory,
-            object_store_memory=object_store_memory,
-            resources=resources,
-            accelerator_type=accelerator_type,
-            max_concurrency=max_concurrency,
-            max_restarts=max_restarts,
-            max_task_retries=max_task_retries,
-            name=name,
-            namespace=namespace,
-            lifetime=lifetime,
-            placement_group=placement_group,
-            placement_group_bundle_index=placement_group_bundle_index,
-            placement_group_capture_child_tasks=(placement_group_capture_child_tasks),
-            runtime_env=new_runtime_env,
-            max_pending_calls=max_pending_calls,
-            scheduling_strategy=scheduling_strategy,
-        )
+        # override original options
+        old_option_dict = self._option_dict.copy()
```

Comment:
IIUC, this `self._option_dict` is for options specified through .remote() right? Maybe it's clear to name them `options_via_remote_fn` and `options_via_options_fn` (or something similar) instead of old and new?